products/impulse/README.md

```diff
@@ -0,0 +1,91 @@
+# helpwave impulse
+
+This document includes all the necessary information to understand the idea and concepts behind helpwave impulse. 
+
+## The idea
+
+The aim of helpwave impulse is to reduce the load on the health care sector by helping people to live a healthier lifestyle. This will reduce the occurrence of illness, which leads to a more sustainable population. Healthier in this context means an improvement of the nutrition, fitness or mental health.
```

Comment:
"this will reduce the burden of disease, which leads to increased public health" - better wording in the medical context. Population is usually reserved for study populations or animal populations.